src/app/src/utils/index.js

```diff
@@ -74,7 +74,10 @@ export class DataIndexer {
     }
 
     formatForViz(key, geo, year) {
-        const resp = this.survey[key];
+        let resp = this.survey[key];
+        if (key.includes('.')) {
+            resp = Object.values(this.survey).find(q => q.qcode === key);
+        }
```

Comment:
Consider adding a comment explaining what this does, and when / why it is needed.

Reply:
Added a comment. For additional reference here, this indicates a Likert scale question, which uses a qcode as an  identifier instead of a key. We can grab the question from any one of the response/question pairs that share the qcode.